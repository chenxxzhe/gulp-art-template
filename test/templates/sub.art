<% extend('./layout.art', { "title": "产品列表" }) %>

<% block('subProduct', function(){ %>
<ul>
  <li>product</li>
</ul>
<% }) %>
