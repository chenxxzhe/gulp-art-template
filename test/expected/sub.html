<div>
  <h1>产品列表</h1>
  
<ul>
  <li>product</li>
</ul>

</div>


